README.md

```diff
@@ -5,3 +5,313 @@ qgepplugin
 This plugin is part of the QGEP Wastewater project for QGIS.
 
 [![Build Status](https://travis-ci.org/QGEP/qgepplugin.svg?branch=master)](https://travis-ci.org/QGEP/qgepplugin)
+
+# QGEP-SWMM
+
+## Installation
+
+### Install QGEP plugin 
+
+* Download the plugin: https://github.com/iggroup/qgepplugin/tree/swmm/src
+* Save in the QGIS plugin path in a folder named `qgepplugin`.
```

Comment:
The official plugin installation guide is here: https://qgep.github.io/docs/installation-guide/workstation.html#install-qgep-plugin

Better to link to this one, so we have one place to update if we ever (finally ;) ) ship the plugin in the main repository